server/package.json

```diff
@@ -4,6 +4,14 @@
   "description": "Jax Pet Names API",
   "main": "index.js",
   "license": "MIT",
+  "scripts": {
+    "db:create": "node knex_create.js",
+    "db:drop": "node knex_drop.js",
+    "db:migrate": "knex migrate:latest",
+    "db:seed": "knex seed:run",
+    "db:reset": "node knex_drop.js && node knex_create.js && knex migrate:latest",
+    "db:setup": "node knex_drop.js && node knex_create.js && knex migrate:latest && knex seed:run"
+  },
```

Comment:
I'm mimicking [Rails](https://guides.rubyonrails.org/active_record_migrations.html) commands for database management here. Open to ideas if there's clearer commands. One or two of these scripts will probably be mapped up to the root level package.json as build scripts for production.

Reply:
Excellent.  I'll take a look and try to get Docker set up and give it a try.